xline-client/src/clients/lease.rs

```diff
@@ -77,22 +79,23 @@ impl LeaseClient {
         Ok(cmd_res.decode().into())
     }
 
-    /// Send `LeaseRevokeRequest` by `CurpClient`
+    /// Revokes a lease. All keys attached to the lease will expire and be deleted.
     ///
     /// # Errors
     ///
-    /// If client failed to send request
+    /// This function will return an error if the inner RPC client encountered a propose failure
```

Comment:
It's "the inner CURP client", not "the inner RPC client". Please keep consistency with other comments.

Reply:
the `revoke` method actually use the lease RPC client and does not use CURP client.